tests/integration/test_vulnerability_detector/test_feeds/archlinux/test_extra_tags_archlinux_feed.py

```diff
@@ -70,15 +124,97 @@ def modify_feed(test_values, request):
 
 
 def test_no_feed_changes(clean_vuln_tables, get_configuration, configure_environment, restart_modulesd):
-    """Check if the feed is imported successfully by default."""
+    '''
+    description: Check if the feed is imported successfully by default. To do this, a vulnerability event is expected
+                 and check if the number of vulnerabilities inserted in the VULNERABILITIES table of CVE DB is the
+                 expected.
+
+    wazuh_min_version: 4.3.0
+
+    parameters:
+        - clean_vuln_tables:
+            type: fixture
+            brief: Clean vulnerabilities tables.
+        - get_configuration:
+            type: fixture
+            brief: Get configurations from the module.
+        - configure_environment:
+            type: fixture
+            brief: Configure a custom environment for testing.
+        - restart_modulesd:
+            type: callable
+            brief: Restart the `wazuh-modulesd` daemon.
+
+    assertions:
+        - Verify that the number of vulnerabilities is equal to the expected number of vulnerabilities inserted in the
+          VULNERABILITIES table of CVE DB.
+
+    input_description:
+        - Test cases are defined in the custom_archlinux_feed.json file. This file contains two different kinds of
+          vulnerabilities. This test gets their configuration of the wazuh_invalid_archlinux_feed.yaml file.
+
+    expected_output:
+        - 'Number of inserted vulnerabilities is not the expected.'
+        - r'Expected, .*, Got: .*'
```

Comment:
Add the `check_feed_imported_successfully` callback
```suggestion
        - r'INFO: .* The update of the .* feed finished successfully.'
```

Reply:
Done in this [link](https://github.com/wazuh/wazuh-qa/commit/92b4e92a92d158ca4a5355cb342855605b51e726)